xml/templates/civicrm_sample.tpl

```diff
@@ -137,9 +137,9 @@ INSERT INTO `civicrm_contact_type`
 INSERT INTO `civicrm_dashboard`
     ( `domain_id`, `name`, `label`, `url`, `permission`, `permission_operator`, `column_no`, `is_minimized`, `is_active`, `weight`, `is_fullscreen`, `fullscreen_url`)
     VALUES
-    ( @domain_id, 'report/7', '{ts escape="sql"}Donor Summary{/ts}'       , 'civicrm/report/instance/7?reset=1&section=2&snippet=5&charts=barChart',  'access CiviContribute', 'AND', 0, 0,'1', 4, '1', 'civicrm/report/instance/7?reset=1&section=2&snippet=5&charts=barChart&context=dashletFullscreen'),
+    ( @domain_id, 'report/7', '{ts escape="sql"}Donor Summary{/ts}'       , 'civicrm/report/instance/7?reset=1&section=1&snippet=5&charts=barChart',  'access CiviContribute', 'AND', 0, 0,'1', 4, '1', 'civicrm/report/instance/7?reset=1&section=1&snippet=5&charts=barChart&context=dashletFullscreen'),
```

Comment:
The change here is in the `section` parameter?

Does anyone know what that means? Should it be an upgrade concern?